Modules/Quotas/src/Quotas.Infrastructure/Persistence/Database/EntityConfigurations/IndividualQuotaEntityTypeConfiguration.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Backbone.Modules.Quotas.Domain.Aggregates.Identities;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+
+namespace Backbone.Modules.Quotas.Infrastructure.Persistence.Database.EntityConfigurations;
+
+public class IndividualQuotaEntityTypeConfiguration : IEntityTypeConfiguration<IndividualQuota>
+{
+    public void Configure(EntityTypeBuilder<IndividualQuota> builder)
+    {
+        builder.HasKey(x => x.Id);
+        builder.Property(x => x.Period);
+        builder.Property(x => x.MetricKey);
+        builder.Property(x => x.Max);
```

Comment:
Can you ignore the `Weight` property, so that it's not accidentally added some day in the future? Do the same for the TierQuotaEntityTypeConfiguration

Reply:
You forgot to ignore the Weight of the TierQuota